pkg/dbal/DbalConsumer.php

```diff
@@ -176,7 +176,7 @@ protected function receiveMessage()
                 ->where('queue = :queue')
                 ->andWhere('(delayed_until IS NULL OR delayed_until <= :delayedUntil)')
                 ->orderBy('priority', 'desc')
-                ->addOrderBy('id', 'asc')
+                ->addOrderBy('id', 'desc')
```

Comment:
Won't this change it from a fifo queue to a lifo queue?

Reply:
I messed up everything. I must've fixed priority ordering not id.